src/vcModals.h

```diff
@@ -22,6 +24,7 @@ enum vcModalTypes
   vcMT_ConvertAdd,
   vcMT_ConvertOutput,
   vcMT_ConvertTempDirectory,
+  vcMT_screenshot,
```

Comment:
This is inconsistent with the other modal names (but I would encourage removing the modal anyway).